src/Bundle/CoreBundle/Model/StatePlace.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: stefankamsker
+ * Date: 19.09.18
+ * Time: 15:16
+ */
+
+namespace UniteCMS\CoreBundle\Model;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * We use this model only for validation!
+ */
+class StatePlace
+{
+    /**
+     * @var string
+     * @Assert\Type(type="string", message="workflow_invalid_place")
+     * @Assert\NotBlank(message="workflow_no_place")
+     */
+    private $identifier;
+
+    /**
+     * @var string
+     * @Assert\Type(type="string", message="workflow_invalid_place")
+     * @Assert\NotBlank(message="workflow_invalid_place")
+     */
+    private $label;
+
+    /**
+     * @var string
+     * @Assert\Choice(choices={"", "primary", "notice", "info", "success", "warning", "error", "danger"}, message="workflow_invalid_category")
+     */
+    private $category;

```

Comment:
Why do we allow an empty choice? I think we should allow `null` or one of the defined categories